### ISSUE TITLE
[5pts] (Front/Back)end: Connect Register and Login Page

### ISSUE THREAD

**Issue by RafaelPiloto10** [Bestagons/quack#121]
**Fill in the blanks**:
As a user, I would like to login/register, so that my account is stored and accessible.

**Describe the solution you'd like:**
- [x] Connect register to backend
- [ ] Connect login to backend
- [ ] Use database to start storing login/register pages

**Definition of Done - Check as you go!**
- [ ] Tasks have been implemented
- [x] Documented
- [ ] Passing Tests for each task
- [ ] Pull Request made
- [ ] Code Reviewed

**Comment by brendacano** [comment]
@RafaelPiloto10  I was looking at your [PR](#125 ), and since it is restructuring some things I was thinking it would be good to wait so I could restructure what I have so far for the [login ticket](https://github.com/Bestagons/quack/issues/7). Then it could match up a lot easier.

**Comment by Foxworth22** [comment]
How soon will this be complete‽

**Comment by brendacano** [comment]
> How soon will this be complete‽

I think this was already done for the register page. He demod it last week if Im not mistaken

**Comment by RafaelPiloto10** [comment]
To a degree. Currently blocked by #137
